src/networks/beta-3.md

```diff
@@ -20,13 +20,13 @@ Goerli block explorer: [https://goerli.etherscan.io/](https://goerli.etherscan.i
 
 🔍 Block explorer - A block explorer (still heavily in development) is available at [https://fuellabs.github.io/block-explorer-v2/](https://fuellabs.github.io/block-explorer-v2/). Be sure to select `beta-3` from the dropdown on the top right.
 
-Join the [Fuel Labs Discord](https://discord.com/invite/fuelnetwork) and head to the 🧪︱testnet channel to get support from our team.
+Join the [Fuel Labs Forum](https://forum.fuel.network/) to get support from our team and others building on Fuel!
 
 ## SDK Versioning
 
 Version 0.38.0 or higher is the recommended version of the TS SDK on `beta-3`.  
 
-Version 0.39.0 or higher is the recommended version for the Rusk SDK on `beta-3`.
+Version 0.39.0 or higher is the recommended version for the Rust SDK on `beta-3`.
```

Comment:
can we also specify an upper version limit here?

Reply:
Yes good call will stop at current version for beta-4